### ISSUE TITLE
Is there a way to output queries to the console?

### ISSUE THREAD

**Issue by andrerpena** [robconery/massive-js#282]
I just migrated from Bookshelf to Massive and incredibly some queries got much slower now. That was surprising because I consider Massive to be lighter, simpler and better designed. I believe there's some problem in my architecture/migration.

Can I log queries generated by Massive? I know I can use PgAdmin for that, but it would be much more convenient to do that on the console.

Thank you.

**Comment by robconery** [comment]
Weird. Would you mind sharing the queries? Are you using caching of any kind?

We don't have logging just yet - but are looking at dropping it in (we've had multiple requests).

Also - can you show me how you're using Massive?

**Comment by andrerpena** [comment]
I found my problem. I'm so embarrassed.

Actually, during my `Bookshelf -> Massive` transition I changed 1 `like` to `ilike`. This simple change makes the query 6 to 7 slower.

**datasource**: 1.040.000 contacts intentionally unindexed.
**query**: select * from contact where name ilike 'andre' (+ equivalent count for paging)
**time**: 233ms (like) or 1600ms (ilike)

Thank you @robconery . Anyway, I think the logging will be very welcome.

For those coming from Google: **Massive is not slower than Bookshelf**.
